interfaces/builtin/home.go

```diff
@@ -74,6 +74,10 @@ owner @{HOME}/snap/ r,
 # files; only allow writes to files, not the mount point).
 owner /run/user/[0-9]*/gvfs/{,**} r,
 owner /run/user/[0-9]*/gvfs/*/**  w,
+
+# Disallow writes to the well-known directory included in
+# the user's PATH on several distributions
+deny @{HOME}/bin/{,**} wl,
```

Comment:
I still like 'audit deny' since I'd like to know when an application tried to do this, but that is not a blocker. +1

Reply:
+1 I will add the audit, I forgot that deny stopped auditing